### PR TITLE
Time sheet UI

### PR DESCRIPTION
**Pull request by nguyennls**
add floaty button
redmine: https://dev.framgia.com/redmine/issues/138712
effect: none

### CODE REVIEW

File: src/wsm/wsm/viewcontrollers/TimeSheetViewController.swift [hunk 1]
```diff
@@ -7,26 +7,35 @@
 //
 
 import UIKit
+import Floaty
+import InAppLocalize
 
 class TimeSheetViewController: BaseViewController {
 
+    var floaty = Floaty()
```
Comment (daolq3012) [inline src/wsm/wsm/viewcontrollers/TimeSheetViewController.swift]: -> private let floaty = Floaty()
Chuẩn hơn a nhể

Comment (nguyennls) [reply src/wsm/wsm/viewcontrollers/TimeSheetViewController.swift]: updated

File: src/wsm/wsm/extensions/Floaty.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+//
+//  Created by framgia on 9/25/17.
+//  Copyright © 2017 framgia. All rights reserved.
+//
+
+import Foundation
+import Floaty
+
+extension Floaty {
+    func addButton(title: String, hexColor: String, imageName: String = "") {
```
Comment (duongdang152) [inline src/wsm/wsm/extensions/Floaty.swift]: should use 
func addButton(title: String, hexColor: String, image:UIImage? = nil)

Comment (nguyennls) [reply src/wsm/wsm/extensions/Floaty.swift]: update

**Pull request merged**